.drone.yml

```diff
@@ -56,7 +56,7 @@ pipeline:
       event: [ push, tag, pull_request ]
 
   build:
-    image: webhippie/golang:edge
+    image: webhippie/golang:1.8
```

Comment:
why not use `edge` in build step?

Reply:
Releases are maid under https://github.com/go-gitea/gitea/blob/ce7af57f8adc7b879e85e8deac6235038724da48/.drone.yml#L185 that should use the latest version? I think the goal is to validate that it compile under 1.8.

Reply:
Yes but this but binary built in this step is used in docker images

Reply:
There is now step to test if it builds without gcc so that could be reused to test if it builds in 1.8

Reply:
With #2927 will be build inside first stage of docker ^^.
In the meantime, the binary is re-build inside a container before (that should override this step). https://github.com/go-gitea/gitea/blob/master/docker/Makefile#L10
From my point either edge or 1.8 is good since test stage will also check that the binary will compile.